rust/python_bindings/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "rust_bindings"
```

Comment:
@HammadB what do you think about keeping the folder name and the crate name in sync
e.g. rust_bindings vs python_bindings here
I like that the Python import is currently `rust_bindings` but dislike that these are named separately so could go either way

Reply:
agreed